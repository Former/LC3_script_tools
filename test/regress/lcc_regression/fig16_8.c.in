355
3
45
12
4
345
45
35
34
1145
355
45
35
34
145
3
45
12
4
345

