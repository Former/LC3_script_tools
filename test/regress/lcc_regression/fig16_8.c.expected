Input Exam 1 score for student 0 : 355
Input Exam 1 score for student 1 : 3
Input Exam 1 score for student 2 : 45
Input Exam 1 score for student 3 : 12
Input Exam 1 score for student 4 : 4
Input Exam 1 score for student 5 : 345
Input Exam 1 score for student 6 : 45
Input Exam 1 score for student 7 : 35
Input Exam 1 score for student 8 : 34
Input Exam 1 score for student 9 : 1145

Input Exam 2 score for student 0 : 355
Input Exam 2 score for student 1 : 45
Input Exam 2 score for student 2 : 35
Input Exam 2 score for student 3 : 34
Input Exam 2 score for student 4 : 145
Input Exam 2 score for student 5 : 3
Input Exam 2 score for student 6 : 45
Input Exam 2 score for student 7 : 12
Input Exam 2 score for student 8 : 4
Input Exam 2 score for student 9 : 345

Total for Student 0 = 710
Total for Student 1 = 48
Total for Student 2 = 80
Total for Student 3 = 46
Total for Student 4 = 149
Total for Student 5 = 348
Total for Student 6 = 90
Total for Student 7 = 47
Total for Student 8 = 38
Total for Student 9 = 1490
